features/shopcarts.feature

```diff
@@ -111,4 +111,22 @@ Scenario: List all shopcarts
     And I should see "1.99" in the results in "Shopcart" page
     And I should not see "Lillisluo" in the results in "Shopcart" page
     And I should not see "Pear" in the results in "Shopcart" page
-    And I should not see "199.99" in the results in "Shopcart" page
\ No newline at end of file
+    And I should not see "199.99" in the results in "Shopcart" page
+
+Scenario: List all items under a shopcart
+    When I visit the "Shopcart" api page
+    And I set the "Name" to "Wan-Yu" in "Shopcart" page
+    And I press the "Search" button in "Shopcart" page
+    Then I should see the message "Success"
+    When I copy the "Id" field in "Shopcart" page
+    And I visit the "Item" api page
+    And I set the "Shopcart ID" within clipboard in "Item" page
+    And I press the "Search" button in "Item" page
+    Then I should see the message "Success"
+    And I should see "Apple" in the results in "Item" page
+    And I should see "1" in the results in "Item" page
+    And I should see "1.99" in the results in "Item" page
+    And I should see "Mango" in the results in "Item" page
+    And I should see "Orange" in the results in "Item" page
+    And I should not see "Peach" in the results in "Item" page
+    And I should not see "100" in the results in "Item" page
```

Comment:
question 🙋‍♀️ what does "100" refer to here?

Reply:
"100" is just a number that should be excluded from the feature file, this line can be deleted